src/watchlists/EditControls.jsx

```diff
@@ -16,7 +30,28 @@ const EditorControls = ({ checkedItems, isInEditMode, toggleEditMode }) => {
         </Text>
       </TouchableHighlight>
       <View style={styles.editorModeRight}>
-        <TouchableHighlight style={styles.editorButton} underlayColor="#ffebeb">
+        <TouchableHighlight
+          style={styles.editorButton}
+          underlayColor="#ffebeb"
+          onPress={() => {
```

Comment:
lets create a method for this and pass it from the parent component, so you will not need to pass the watchlistId and watclistName